bigbluebutton-tests/playwright/core/elements.js

```diff
@@ -121,7 +121,7 @@ exports.currentSlideBreakoutOption = 'option[data-test="currentSlideBreakoutOpti
 exports.chatBox = 'textarea[id="message-input"]';
 exports.chatMessageItem = 'div[data-test="chatMessageItem"]';
 exports.partnerDisconnectedMessage = 'span[data-test="partnerDisconnected"]';
-exports.chatButton = 'button[data-test="chatButton"]';
+exports.chatButton = 'div[data-test="chatButton"]';
 exports.messagesSidebarButton = 'div[data-test="chatButton"]';
```

Comment:
use only one element + edit data-test value on the component to follow v3.1 pattern: `messagesSidebarButton`

```suggestion
exports.messagesSidebarButton = 'div[data-test="messagesSidebarButton"]';
```